textproc/recode/Portfile

```diff
@@ -24,16 +24,26 @@ long_description    This recode program has the purpose of converting files \
 
 github.tarball_from releases
 
-depends_test        port:py27-cython
+set python.version  37
+set python.branch   [string range ${python.version} 0 end-1].[string index ${python.version} end]
+set python.prefix   ${frameworks_dir}/Python.framework/Versions/${python.branch}
+set python.bin      ${python.prefix}/bin/python${python.branch}
+
+depends_build       port:python${python.version}
 
 depends_lib         port:gettext \
                     port:libiconv
 
+depends_test        port:py${python.version}-cython
+
 if {[string match *gcc* ${configure.compiler}]} {
     patchfiles-append \
-                    pragma.patch
+        pragma.patch
```

Comment:
Please leave the second column aligned at 20 spaces. Same goes for the `configure.env-append` and `test.env` lines below.

Reply:
simply, it doesn't respect the coding style (header); changed.